src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,18 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
+# TODO: need a purl for this?
+.PHONY: mondo-mappings
+MONDO_MAPPINGS_TSV = 'https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv'
+mondo-mappings:
+	wget ${MONDO_MAPPINGS_TSV} -P ../analysis/excluded_terms_in_mondo/ignored/mappings/
+
+# TODO: Need to add dependency for sig/mirror files?
+.PHONY: excluded_terms_in_mondo
+excluded-terms-in-mondo: mondo-mappings
```

Comment:
Since this is running on all ontologies and not just one, the natural thing to do seemed to be to not include something like `component_signature-%.tsv` and `mirror_signature-%.tsv` as makefile recipe dependencies. Instead, these file patterns and their directory are stored in the config for the Python file. Let me know if you wan this refactored in any way.

Reply:
Resolving. This is redundant with this discussion: https://github.com/monarch-initiative/mondo-ingest/pull/35#discussion_r930526547

Reply:
changed